crates/matrix-sdk/src/room/timeline/event_handler.rs

```diff
@@ -277,6 +277,7 @@ fn update_fully_read_item(
             *fully_read_event_in_timeline = false;
         }
         (Some(from), Some(to)) => {
+            *fully_read_event_in_timeline = true;
```

Comment:
By the way, should we check the existing value of this before even looking for `old_idx`?

Reply:
No because we want to be able to update the marker when the fully read event changes.

Reply:
Maybe you misunderstood, I meant can't we set `old_idx` to `None` without iterating the timeline items if `*fully_read_event_in_timeline` is `false`?

Reply:
No because then we can't move the read marker as we don't have its current position?

Reply:
Ohhh, this is for whether the event is in the timeline, not whether the marker exists. Got it.

Reply:
By the way, did you see my message in the matrix room?